config/initializers/raven.rb

```diff
@@ -0,0 +1,7 @@
+if Rails.env.production? || ENV['FORCE_ENABLE_SENTRY'] == 'true'
```

Comment:
Please add note about `FORCE_ENABLE_SENTRY` to the README.  README might need a new configuration section.